helpers/addresses.py

```diff
@@ -659,6 +659,10 @@ def checksum_address_dict(addresses):
     return checksummed
 
 
+with open('helpers/chaindata.json') as chaindata:
+    chain_ids = json.load(chaindata)
+
+
```

Comment:
really clean solution, i like it a lot!

the only thing im wondering here is why put this in addresses.py? id suggest to move this to chaindata.py, and then `from helpers.chaindata import labels` or something.

also maybe instead of str: str set the json up as int: str?